javascript/javascript2/week1/spiritanimal.js

```diff
@@ -0,0 +1,30 @@
+let animals=['butterfly','lion','tiger','fox','cat','parrot','peacock'];
+let textboxValue=document.getElementById('name');
+let printingArea=document.getElementById('print');
+let submit=document.getElementById('submitButton');
+let index = animals[Math.floor(Math.random() * animals.length)];
+let newAnimalBUtton=document.getElementById('newSpiritAnimal');
+submit.addEventListener("click",getSpiritAnimal);
+newAnimalBUtton.addEventListener("click",getNewSpiritAnimal);
+function getSpiritAnimal(){
+    let username = textboxValue.value;
+    if(username == ""||!(isNaN(username))){
```

Comment:
I dont think the isNaN is necessary, empty string check should be good enough

Reply:
getSpiritAnimal could reasonably be named setSpiritAnimal since it does not return a value, it merely sets the value of innerHTML for an element.